processor/resourcedetectionprocessor/README.md

```diff
@@ -78,6 +78,12 @@ ec2:
     * deployment.environment
     * service.instance.id
     * service.version
+
+* Amazon EKS
+
+    * cloud.provider ("aws")
+    * cloud.infrastructure_service ("aws_eks")
+    * k8s.cluster.name (name of the EKS cluster)
```

Comment:
For some reason I thought k8s doesn't have the concept of a cluster name but I see it in the spec, nice

Is now a good time to add arn? If not no worries

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/cloud_provider/aws/eks.md

Reply:
Would rather do separately.